src/network/networkd-manager.c

```diff
@@ -85,7 +85,7 @@ static int match_prepare_for_sleep(sd_bus_message *message, void *userdata, sd_b
         if (b)
                 return 0;
 
-        log_debug("Coming back from suspend, resetting all connections...");
+        log_debug("Coming back from sleep, resetting all connections...");
```

Comment:
I think it's better to keep "suspend" here. "sleep" is the kernel name, but it is very ambiguous. We (and pretty much everybody else) calls this suspend. We have `systemctl suspend`, `suspend-then-hibernate`, etc.